src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import Model.Product;
+
+public class Main {
+    public static void main(String[] args) {
+
+        // ProductListe
+        Product product1 = new Product(1, "Shoe");
+        Product product2 = new Product(2, "Camera");
+        Product product3 = new Product(3, "Laptop");
+
```

Comment:
Aktuell ist das in der Main Methode toter Code, da du ihn nicht benötigst.
Willst du hier noch weitercoden?
Wenn nicht, dann kannst du das vorhandene gerne löschen